archetypes/helidon/src/main/archetype/se/custom/observability.xml

```diff
@@ -21,6 +21,12 @@
                   xsi:schemaLocation="https://helidon.io/archetype/2.0 https://helidon.io/xsd/archetype-2.0.xsd">
     <output>
         <model if="${tracing}">
+            <list key="Main-helidon-imports">
+                <value>io.helidon.webserver.WebTracingConfig</value>
+            </list>
+            <list key="Main-routingBuilder">
```

Comment:
This is not sufficient - the tracer must be created and (ideally) registered with the server:
```
WebServer.builder()
  ...
  .tracer(TracerBuilder.create(config.get("tracing"))
                    .build())
```